homeassistant/const.py

```diff
@@ -614,15 +623,22 @@ class Platform(StrEnum):
 CONCENTRATION_PARTS_PER_BILLION: Final = "ppb"
 
 # Speed units
-SPEED_MILLIMETERS_PER_DAY: Final = "mm/d"
 SPEED_FEET_PER_SECOND: Final = "ft/s"
-SPEED_INCHES_PER_DAY: Final = "in/d"
 SPEED_METERS_PER_SECOND: Final = "m/s"
-SPEED_INCHES_PER_HOUR: Final = "in/h"
 SPEED_KILOMETERS_PER_HOUR: Final = "km/h"
 SPEED_KNOTS: Final = "kn"
 SPEED_MILES_PER_HOUR: Final = "mph"
 
+SPEED_MILLIMETERS_PER_DAY: Final = "mm/d"
+"""Deprecated: please use PRECIPITATION_MILLIMETERS_PER_DAY"""
+
+SPEED_INCHES_PER_DAY: Final = "in/d"
+"""Deprecated: please use PRECIPITATION_INCHES_PER_DAY"""
+
+SPEED_INCHES_PER_HOUR: Final = "in/h"
+"""Deprecated: please use PRECIPITATION_INCHES_PER_HOUR"""
+
+
```

Comment:
@emontnemery should these constants be removed completely? or should they be deprecated?

Reply:
I have about it, and I think it's better to deprecate for now - less risky.